frontend/deploy/run.sh

```
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+base=/OJ_FE
+
+build_vendor_dll()
+{
+  if [ ! -e "${base}/build/vendor-manifest.json" ]
+  then
+      npm run build:dll
+  fi
+}
+cd $base
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)